tests/daemon/test-persistent-cache.c

```diff
@@ -1186,12 +1156,12 @@ main (gint                argc,
                        test_persistent_cache_does_not_compute_offset_when_boot_id_is_same);
   ADD_CACHE_TEST_FUNC ("/persistent-cache/reads-cached-boot-offset",
                        test_persistent_cache_reads_cached_boot_offset);
-  ADD_CACHE_TEST_FUNC ("/persistent-cache/get-offset-wont-update-timestamps-if-it-isnt-supposed-to",
-                       test_persistent_cache_get_offset_wont_update_timestamps_if_it_isnt_supposed_to);
   ADD_CACHE_TEST_FUNC ("/persistent-cache/updates-timestamps-on-finalize",
                        test_persistent_cache_updates_timestamps_on_finalize);
   ADD_CACHE_TEST_FUNC ("/persistent-cache/get-offset-can-build-boot-metadata-file",
-                       test_persistent_cache_get_offset_can_build_boot_metadata_file);
+                       test_persistent_cache_can_build_boot_metadata_file);
+  ADD_CACHE_TEST_FUNC ("/persistent-cache/reads-cached-boot-offset-when-initialized",
```

Comment:
nit: If we keep this test, it would be nice to list it in the same order that it appears above (right below `test_persistent_cache_reads_cached_boot_offset`).